Gemfile

```diff
@@ -52,9 +52,11 @@ end
 
 
 group :test do
-  gem "simplecov"
-  gem "codeclimate-test-reporter", "~> 1.0.0"
-  gem "capybara"
+  gem 'simplecov'
+  gem 'codeclimate-test-reporter', '~> 1.0.0'
+  gem 'capybara'
+  gem 'capybara-selenium'
```

Comment:
Should explicit versioning be used - not sure if we have a policy in place?

Reply:
I like Thoughtbot's approach:

```
* Use a versionless Gemfile declarations for gems that are safe to update often, such as pg, thin, and debugger.

* Use an exact version in the Gemfile for fragile gems, such as Rails.
```

https://github.com/thoughtbot/guides/tree/master/best-practices#bundler

Reply:
OK good advice thanks